impl/src/rendering/templates/partials/targeted_dependencies.template

```diff
@@ -1,8 +1,8 @@
 {% for targeted_dep in crate.targeted_deps %} + selects.with_or({
         # {{ targeted_dep.target }}
         (
-{%- for condition in targeted_dep.conditions %}
-            "{{ condition }}",
+{%- for platform_target in targeted_dep.platform_targets %}
+          "@rules_rust//rust/platform:{{platform_target}}",
```

Comment:
This should use a templated workspace name. ie:
https://github.com/google/cargo-raze/blob/eee9428916139e4329d25d5b1c42de120486463f/impl/src/rendering/templates/crate.BUILD.template#L10

Reply:
yep noted when I was updating `Cargo.toml` to the latest

Reply:
```suggestion
          "@{{ rust_rules_workspace_name }}//rust/platform:{{platform_target}}",
```